python/iceberg/core/filesystem/filesystem_tables.py

```diff
@@ -36,15 +35,15 @@ def load(self, location):
 
         return BaseTable(ops, location)
 
-    def create(self, schema, table_identifier=None, spec=None, properties=None, location=None):
+    def create(self, schema, table_identifier=None, spec=None, properties=None):
```

Comment:
I agree with using table_identifier here. Should we also rename it in `load` since the location is the table identifier for FS tables?